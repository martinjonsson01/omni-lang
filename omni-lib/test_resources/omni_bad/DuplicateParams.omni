module DuplicateParams()

function(x: Int, x: Int) : Unit = ()

main() : Int = 0