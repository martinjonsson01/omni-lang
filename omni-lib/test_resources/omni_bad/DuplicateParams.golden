error:
duplicate identifier 'x' at test_resources\omni_bad\DuplicateParams.omni:3:10-3:11
                            test_resources\omni_bad\DuplicateParams.omni:3:18-3:19
     